README.md

```diff
@@ -58,7 +58,9 @@ Per default, the server will use the `LOCAL_FILE_READ_ONLY` storage type and ser
 ```
 $ docker run -p 8080:80 -v ./storage2:/storage2 -e API_BASE_PATH=/api/v3.1 -e STORAGE_TYPE=LOCAL_FILE_BACKEND -e STORAGE_PATH=/storage2 basyx-python-sdk-http-server
 ```
+## Acknowledgments
 
+This project uses code from the [tiangolo/uwsgi-nginx-docker][10] repository. Feel free to check out their project for more information on running uWSGI and Nginx in Docker.
```

Comment:
I would call this: "This Dockerfile is inspired by ...", because otherwise it is unclear why we didn't fork. I would also remove the second sentence.